spec/helpers/application_helper_spec.rb

```diff
@@ -1,19 +1,15 @@
 require "spec_helper"
-include ActionView::Helpers::JqueryHelper
-include JsHelper
-include ToolbarHelper
```

Comment:
TBH I don't remember *what* was breaking here, but something was. It seemed easier to fix the helper specs than investigate the problem.